images/terraform/main.tf

```
@@ -8,32 +8,66 @@ variable "target_repository" {
   description = "The docker repo into which the image and attestations should be published."
 }
 
-module "latest-config" {
+module "pre-busl-config" {
   source         = "./config"
   extra_packages = ["terraform<1.6"]
 }
 
-module "latest" {
+module "latest-busl-config" {
+  source         = "./config"
+  extra_packages = ["terraform"]
+}
+
+module "pre-busl" {
   source            = "../../tflib/publisher"
   name              = basename(path.module)
   target_repository = var.target_repository
-  config            = module.latest-config.config
+  config            = module.pre-busl-config.config
   build-dev         = true
 }
 
-module "test-latest" {
+module "test-pre-busl" {
   source = "./tests"
-  digest = module.latest.image_ref
+  digest = module.pre-busl.image_ref
+}
+
+module "latest-busl" {
+  source            = "../../tflib/publisher"
+  name              = basename(path.module)
+  target_repository = var.target_repository
+  config            = module.latest-busl-config.config
+  build-dev         = true
 }
 
-resource "oci_tag" "latest" {
-  depends_on = [module.test-latest]
-  digest_ref = module.latest.image_ref
+module "test-latest-busl" {
+  source = "./tests"
+  digest = module.latest-busl.image_ref
+}
+
+resource "oci_tag" "latest-busl" {
+  depends_on = [module.test-latest-busl]
+  digest_ref = module.latest-busl.image_ref
   tag        = "latest"
 }
 
-resource "oci_tag" "latest-dev" {
-  depends_on = [module.test-latest]
-  digest_ref = module.latest.dev_ref
+resource "oci_tag" "latest-busl-dev" {
+  depends_on = [module.test-latest-busl]
+  digest_ref = module.latest-busl.dev_ref
   tag        = "latest-dev"
 }
+
+locals { pre_busl_tags = toset(["1.5.7", "1.5"]) }
```

```suggestion
locals { pre_busl_tags = toset(["1.5.7", "1.5", "latest-busl", "latest-busl-dev"]) }
```

would these be useful? I imagine most folks don't know or want to know that 1.5.7 is the latest prebusl

---

I guess these need auth exceptions so don't hold up on this comment!

---

This might be useful, but these are the _pre_-busl images. 

Maybe `latest-non-busl(-dev)`?

wdyt @dlorenc

---

maybe latest-mpl?